app/Models/Contract.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Contract extends Model
+{
+    public function requests() {
+        return $this->belongsTo(Request::class);
+    }
+
+    public function contractDrivers() {
+        return $this->hasOne(ContractDriver::class);
+    }
+
+    public function users() {
+        return $this->belongsTo(User::class, "supplier_id");
```

Comment:
```suggestion
        return $this->belongsTo(User::class, 'supplier_id');
```
sửa hết các phần tương tự

Reply:
vâng ạ